engine/gamesys/src/gamesys/resources/res_material.cpp

```diff
@@ -101,6 +101,9 @@ namespace dmGameSystem
 
     static dmResource::Result AcquireResources(dmResource::HFactory factory, dmRenderDDF::MaterialDesc* ddf, MaterialResources* resources)
     {
+        memset(resources->m_Textures, 0, sizeof(resources->m_Textures));
+        memset(resources->m_SamplerNames, 0, sizeof(resources->m_SamplerNames));
```

Comment:
Had to move these, otherwise we try to release junk texture pointers if the material can't be loaded.